src/components/charts/PieChart.vue

```diff
@@ -0,0 +1,79 @@
+<template>
+  <div class="pie-chart-container">
+    <Pie
+      id="my-pie-chart-id"
+      :options="chartOptions"
+      :data="chartData"
+    />
+  </div>
+</template>
+
+<script>
+import { Pie } from 'vue-chartjs';
+import { Chart as ChartJS, Title, Tooltip, Legend, ArcElement } from 'chart.js';
+
+ChartJS.register(Title, Tooltip, Legend, ArcElement);
+
+export default {
+  name: 'PieChart',
+  components: { Pie },
```

Comment:
Add a prop section here with called hospital which is required and takes an object eg
```js
    props: {
        hospitals: {
            type: Array,
            required: true
        },
   }
```